test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -471,10 +472,22 @@ private String summarizeProcessInstanceCreation(final Record<?> record) {
         .append("new <process ")
         .append(formatId(value.getBpmnProcessId()))
         .append(">")
+        .append(summarizeStartInstructions(value.getStartInstructions()))
         .append(summarizeVariables(value.getVariables()))
         .toString();
   }
 
+  private String summarizeStartInstructions(
+      final List<ProcessInstanceCreationStartInstructionValue> startInstructions) {
+    if (startInstructions.isEmpty()) {
+      return " (default start) ";
```

Comment:
❓ Do we need to highlight this? I'd rather reduce the data and only highlight the alternative case. WDYT?

Reply:
I am strongly in favor of being explicit. My goal is that a new engineer who just joined the team can read the log and have an intuitive understanding of it. 

What I want to avoid is that someone is wondering: Were there start instructions? Would this be printed in the log? and then they need to look up the code to make sense of the log output.

With it being explicit, an engineer who doesn't know about start instructions yet could speculate that there might be other ways to start than the default way. And an engineer who does know about start instructions can read that they were not used.